.gitignore

```diff
@@ -48,4 +48,6 @@ airwallex/gen
 airwallex/bin
 example/target
 example/gen
-example/bin
\ No newline at end of file
+example/bin
+
+*.properties
```

Comment:
Will this only affect .properties in root directory or all .properties files? Ideally we want to limit to the specific file that contains keys.

Reply:
only affect .properties in root directory